Resources/Textures/Mobs/Species/carp_holo.rsi/meta.json

```diff
@@ -0,0 +1,103 @@
+{
+  "version": 1,
+  "size": {
+    "x": 32,
+    "y": 32
+  },
+  "license": "CC-BY-SA-3.0",
+  "copyright": "Taken from https://github.com/tgstation/tgstation",
```

Comment:
It'd be a good idea to put the commit at which you took these sprites at here and on all RSI you added.